### ISSUE TITLE
web: `/home` raises error happeningly

### ISSUE THREAD

**Issue by kino-ma** [nkzwlab/2021f-group-9#31]
Maybe the SSR causes

```
pages/home.tsx (112:10) @ _callee$

  110 | 
  111 |     // Unknown errors
> 112 |     throw new Error("エラーが発生しました");
      |          ^
  113 |   }
  114 | 
  115 |   const threadsResp = await get("/thread");
```

**Comment by kino-ma** [comment]
If I reload the page, this error disappears. So maybe the cause is related to SSR.

**Comment by kino-ma** [comment]
Closed by #34
